src/lib/mina_caqti/README.md

```diff
@@ -0,0 +1,76 @@
+Mina_caqti
+==========
+
+This library is designed to assist in querying relational databases
+using the Caqti library. It is used extensively for querying the
+archive database in the modules `Processor` and `Load_data` modules in
+`Archive_lib`.
```

Comment:
```suggestion
archive database in the `Processor` and `Load_data` modules in
`Archive_lib`.
```

Reply:
@ghost-not-in-the-shell  do we mean `src/app/archive/lib/archive_lib.opam` ?  would a link or file path help here? (It would help me)